pkg/sbom/sbom.go

```diff
@@ -47,15 +65,17 @@ func SBOMWorkflow(
 		sbomDocs = append(sbomDocs, newData(depGraph, service.MimeTypeCycloneDXJSON, sbomBytes))
 	}
 
+	logger.Printf("Successfully generated SBOM documents: count %d\n", len(sbomDocs))
```

Comment:
How about "Successfully generated %d SBOM documents" or even better "Successfully generated %d SBOMs"?

Reply:
we had that at first 😄 but there was no easy way to do proper pluralization without pulling in more dependencies (i.e. combat thinks like `generate 1 SBOMs`) so we went with this admittedly awkward output.

Reply:
Understood. I usually would do something like "Successfully generated %d SBOM document(s)" or even just live with the grammatical awkwardness. Not a big deal though.